src/map-interactions.react.js

```diff
@@ -206,6 +214,7 @@ export default class MapInteractions extends Component {
   /* eslint-disable complexity, max-statements */
   @autobind
   _onWheel(event) {
+    event.stopPropagation();
```

Comment:
We had an issue and a subsequent PR to remove this `stopPropagation` a couple of weeks ago. Is there a reason why propagation needs to be stopped here?